src/main/java/net/sf/jsqlparser/parser/ParserKeywordsUtils.java

```diff
@@ -0,0 +1,429 @@
+/*-
+ * #%L
+ * JSQLParser library
+ * %%
+ * Copyright (C) 2004 - 2021 JSQLParser
+ * %%
+ * Dual licensed under GNU LGPL 2.1 or Apache License 2.0
+ * #L%
+ */
+package net.sf.jsqlparser.parser;
+
+import org.javacc.jjtree.JJTree;
+import org.javacc.parser.JavaCCGlobals;
+import org.javacc.parser.JavaCCParser;
+import org.javacc.parser.RCharacterList;
+import org.javacc.parser.RChoice;
+import org.javacc.parser.RJustName;
+import org.javacc.parser.ROneOrMore;
+import org.javacc.parser.RSequence;
+import org.javacc.parser.RStringLiteral;
+import org.javacc.parser.RZeroOrMore;
+import org.javacc.parser.RZeroOrOne;
+import org.javacc.parser.RegularExpression;
+import org.javacc.parser.Semanticize;
+import org.javacc.parser.Token;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InvalidClassException;
+import java.nio.charset.Charset;
+import java.nio.charset.CharsetEncoder;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ParserKeywordsUtils {
+    public final static CharsetEncoder CHARSET_ENCODER = StandardCharsets.US_ASCII.newEncoder();
+
+    public final static int RESTRICTED_FUNCTION = 1;
```

Comment:
While this works, Java's `enum` combined with `EnumSet.of()` combined is less 'Hacky'?

Reply:
for example `enum Restricted { FUNCTION, SCHEMA, TABLE, COLUMN ... }`